### PR TITLE
Add issue form template for Helm Chat

### PR DESCRIPTION
**Pull request by potiuk**
With so many people reviewing nobody noticed that we forgot to add
Helm Chart issue form :)

This rectifies the mistake.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

**Comment by potiuk** [comment]
Totally, Totally forgot it 🤦. You can try it here:  https://github.com/potiuk/airflow/issues/new/choose

**Comment by mik-laj** [comment]
Should we add feature request form?

**Comment by potiuk** [comment]
> Should we add feature request form?

We have one generic form for feature. We did not separate it per providers and I think we should not do it for Helm Chart either.

**Comment by potiuk** [comment]
Screenshot 2021-08-30 20 12 12

File: .github/ISSUE_TEMPLATE/airflow_helmchart_bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+        You can open [Pull Request](https://github.com/apache/airflow/pulls) immediately instead.
+        <br clear='left'/>"
+      # yamllint enable rule:line-length
+  - type: dropdown
+    attributes:
+      label: Official Helm Chart version
+      description: >
+        What Apache Airflow version are you using? Only Airflow 2 is supported for bugs. If you wish to
+        discuss Airflow 1.10, open [Discussion](https://github.com/apache/airflow/discussions) instead!
```
Comment (jedcunningham) [inline .github/ISSUE_TEMPLATE/airflow_helmchart_bug_report.yml]:
```suggestion
        What Apache Airflow Helm Chart version are you using?
```

**Review by jedcunningham — CHANGES_REQUESTED** [review]

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**